ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -116,6 +116,13 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     operatorframework.io/suggested-namespace: avi-system
     containerImage: projects.packages.broadcom.com/ako/ako-operator@sha256:cfc30e3a06fd3afa54e6a0a10a3aac1604395a60f04b291fcbba61e3529b4876
+    features.operators.openshift.io/disconnected: "true"
```

Comment:
@DixitAakash are we meeting all requirement specified here: https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/operators/developing-operators#olm-enabling-operator-for-restricted-network_osdk-generating-csvs for `disconnected`?

Reply:
@akshayhavile We are meeting all the requirements that are applicable for us. I have also updated the Makefile to  use image digests instead of tags in the images generated as part of bundle. Infact, we have already been using digests in manifests, just that makefile was not updated before to generate manifests with image digests instead of tags.